src/analyze/analyze-security.c

```diff
@@ -1798,16 +1803,14 @@ static int assess(const SecurityInfo *info,
                         if (d)
                                 description = d;
 
-                        if (json_variant_by_key(policy, a->json_field) != NULL)
-                                id = a->json_field;
-
                         r = table_add_many(details_table,
-                                           TABLE_STRING, checkmark,
+                                           TABLE_BOOLEAN_CHECKMARK, checkmark == special_glyph(SPECIAL_GLYPH_CHECK_MARK),
                                            TABLE_SET_MINIMUM_WIDTH, 1,
                                            TABLE_SET_MAXIMUM_WIDTH, 1,
                                            TABLE_SET_ELLIPSIZE_PERCENT, 0,
                                            TABLE_SET_COLOR, color,
-                                           TABLE_STRING, id, TABLE_SET_URL, a->url,
+                                           TABLE_STRING, a->id, TABLE_STRING, a->json_field,
+                                           TABLE_SET_URL, a->url,
```

Comment:
the URL should be assigned to the `id` colun, not the `json_field` column. hence insert the new line *after* the TABLE_SET_URL thing, not before